src/api/app/controllers/api_controller.rb

```diff
@@ -68,18 +68,32 @@ def visualization_2
   def visualization_3
     @fiscal_information = School.joins(:Fiscal, :Fact)
               .select('
-                (local_revenue + state_revenue + other_revenue + fed_revenue) AS revenue,
-                school.state_lea_id,
                 school_name,
+                (local_revenue + state_revenue + other_revenue + fed_revenue) AS totalRevenue,
                 sat_math,
                 sat_reading,
                 sat_writing
               ')
               .where.not(data_school_facts: { sat_math: nil })
               .where.not(data_school_facts: { sat_reading: nil })
               .where.not(data_school_facts: { sat_writing: nil })
-              .order('revenue ASC')
+              .order('totalRevenue ASC')
               .all()
-    json_response(@fiscal_information)
+
+    json_response(visualization_3_custom_json(@fiscal_information))
+  end
+
+  def visualization_3_custom_json(data)
```

Comment:
Remove, give raw json from the database and change format on the UI side instead